includes/class-amp-reader-themes.php

```diff
@@ -0,0 +1,546 @@
+<?php
+/**
+ * Fetches and formats data for AMP reader themes.
+ *
+ * @package AMP
+ * @since 1.6.0
+ */
+
+/**
+ * Class AMP_Reader_Themes.
+ *
+ * @since 1.6.0
+ */
+final class AMP_Reader_Themes {
+	/**
+	 * Formatted theme data.
+	 *
+	 * @since 1.6.0
+	 *
+	 * @var array
+	 */
+	private $themes;
+
+	/**
+	 * The name of the currently active theme.
+	 *
+	 * @since 1.6.0
+	 *
+	 * @var string
+	 */
+	private $current_theme_name;
+
+	/**
+	 * Whether themes can be installed in the current WordPress installation.
+	 *
+	 * @since 1.6.0
+	 *
+	 * @var bool
+	 */
+	private $can_install_themes;
+
+	/**
+	 * The default reader theme. Indicates the plugin-provided classic mode should be used.
```

Comment:
The second sentence in this comment is not clear to me, as it reads as if the property would be a boolean. Is this meant to signify something like "Defaults to the plugin-provided classic mode."?